sdk/eventhub/tests.yml

```diff
@@ -8,8 +8,33 @@ resources:
       endpoint: azure
 
 jobs:
+# Azure Cloud
 - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
   parameters:
     MaxParallel: 6
     ServiceDirectory: eventhub
-    TimeoutInMinutes: 120
\ No newline at end of file
+    TimeoutInMinutes: 120
+    ScenarioName: AzureCloud
+    SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+    EnvVars:
+      AZURE_AUTHORITY_HOST: https://login.microsoftonline.com
```

Comment:
@danieljurek shouldn't this env var be part of the subscription config?

Reply:
Yes. This would be a good candidate for a subscription config type of configuration... We'd need to add the ability to export those environment variables to `New-TestResources.ps1`